src/app/components/tables/simple/terra-simple-table.component.ts

```diff
@@ -69,23 +78,23 @@ export class TerraSimpleTableComponent<D> implements OnChanges
 
     public onRowListChange:EventEmitter<void> = new EventEmitter();
 
-    private _headerCheckbox:{ checked:boolean, isIndeterminate:boolean };
-    private _selectedRowList:Array<TerraSimpleTableRowInterface<D>> = [];
+    protected headerCheckbox:{ checked:boolean, isIndeterminate:boolean };
+    protected _rowList:Array<TerraSimpleTableRowInterface<D>>;
```

Comment:
Better make `rowList` private and use getter/setter properly (see review comment in html template)